com_redhat_kdump/service/installation.py

```
@@ -48,6 +47,35 @@ def __init__(self, kernels, sysroot, kdump_enabled, fadump_enabled, reserved_mem
     def name(self):
         return "Configure kernel parameters for kdump and fadump"
 
+    def get_default_crashkernel(self, fadump_enabled):
+        dump_mode = 'kdump'
+        if fadump_enabled:
+            dump_mode = 'fadump'
+
+        args = {'command': 'kdumpctl',
+                'argv': ['get-default-crashkernel', dump_mode],
+                'root': self._sysroot,
+                'filter_stderr': True}
+
+        ck_val = util.execWithCapture(**args)
+
+        if not ck_val:
+            log.warning("Can't retrieve the default crashkernel value from "
+                        "the installed kexec-tools, try to retrieve it from "
+                        "the installer kexec-tools")
+            del args['root']
+            try:
+                ck_val = util.execWithCapture(**args)
+            except FileNotFoundError:
+                pass
```

Maybe generate a error message here rather than fail silently.